ansys/mapdl/core/_commands/solution/inertia.py

```diff
@@ -53,6 +53,12 @@ def acel(self, acel_x="", acel_y="", acel_z="", **kwargs):
     information.
 
     This command is also valid in /PREP7.
+
+    Examples
+    --------
+    Set global y-acceleration to be 9.81
+
+    >>> mapdl.acel('', 9.81)
```

Comment:
```suggestion
    >>> mapdl.acel(acel_y=9.81)
```
Lets get in the habit of using kwargs rather than inputting empty strings for missing args.  Users of Python will be more familiar with this pattern.